apps/rush-lib/src/data/RushConfiguration.ts

```diff
@@ -173,14 +173,29 @@ export default class RushConfiguration {
 
     // If the version is missing or malformed, fall through and let the schema handle it.
     if (expectedRushVersion && semver.valid(expectedRushVersion)) {
-      if (semver.lt(Rush.version, expectedRushVersion)) {
-        throw new Error(`Unable to load ${rushJsonBaseName} because its RushVersion is`
-          + ` ${rushConfigurationJson.rushVersion}, whereas @microsoft/rush-lib is version ${Rush.version}.`
-          + ` Consider upgrading the library.`);
-      } else if (semver.lt(expectedRushVersion, MINIMUM_SUPPORTED_RUSH_JSON_VERSION)) {
+      // Make sure the requested version isn't too old
+      if (semver.lt(expectedRushVersion, MINIMUM_SUPPORTED_RUSH_JSON_VERSION)) {
```

Comment:
We were talking about introducing a schema/configuration version field in rush.json. I think that would be cleaner. Then we very explicitly increment the version if we make a change.

Reply:
I'll come chat about this

---
In reply to: [183514953](https://github.com/Microsoft/web-build-tools/pull/618#discussion_r183514953) [](ancestors = 183514953)

Reply:
We discussed this and Ian was okay with the approach

---
In reply to: [183515290](https://github.com/Microsoft/web-build-tools/pull/618#discussion_r183515290) [](ancestors = 183515290,183514953)